schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-drawer-container>
+  <mat-drawer mode="side" #drawer [opened]="!isHandset">
```

Comment:
I believe this should be a `mat-sidenav` since it's at the top level. @mmalerba will know best